draft-ietf-lamps-documentsigning-eku.md

```diff
@@ -41,17 +41,17 @@ normative:
 
 --- abstract
 
-{{!RFC5280}} specifies several extended key usages for X.509 certificates. This document defines a general purpose document signing extended key usage for X.509 public key certificates which restricts the usage of the certificates for document signing.
+{{!RFC5280}} specifies several extended key usages for X.509 certificates. This document defines a general purpose document signing extended key usage for X.509 public key certificates which specifies the usage of the certificates for document signing.
 
 --- middle
 
 # Introduction
 
-{{!RFC5280}} specifies several extended key usages for X.509 certificates. In addition,
-several extended key usage had been added{{!RFC7299}} as public OID under the IANA repository.
-While usage of any extended key usage is bad practice for publicly trusted certificates,
-there are no public and general extended key usage explicitly assigned for Document Signing certificates.
-The current practice is to use id-kp-emailProtection, id-kp-codeSigning or vendor defined Object ID
+{{!RFC5280}} specifies several extended key purpose identifiers for X.509 certificates. In addition,
+several extended key purpose identifiers were added{{!RFC7299}} as public OIDs under the IANA PKIX Extended key purpose identifiers arc.
```

Comment:
While we're at it can we add a space between added and the reference to 7299?

Can also r/OID/Object Identifier (OID)